feat: add update notes for v2.1.0 (#89)
kellerkinderDE/devenv-shopware: docs/Update.md

          follows: "nixpkgs"
    ...
  ```

> * Following configurations type have been changed from `string` to `ìnt`: `kellerkinder.httpPort` and `kellerkinder.httpsPort`

Reviewer: ```suggestion
* The following configurations types have been changed from `string` to `ìnt`: `kellerkinder.httpPort` and `kellerkinder.httpsPort`
```

> * Due to stability, we use stable branch `nixos-23.11` instead of `nixos-unstable` in our examples

Reviewer: ```suggestion
* For stability reasons, we use the stable branch `nixos-23.11` instead of `nixos-unstable` in our examples.
```
